<launch>
	<node pkg="fsm_bump_go" type="bumpgo_node" name="BumpGo_Node" output="screen"/>
</launch>